proposals/simd/BinarySIMD.md

```diff
@@ -27,143 +27,170 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
+| `i32x4.lt_u`              |    `0x2f`| -                  |
+| `i32x4.gt_s`              |    `0x30`| -                  |
+| `i32x4.gt_u`              |    `0x31`| -                  |
+| `i32x4.le_s`              |    `0x32`| -                  |
+| `i32x4.le_u`              |    `0x33`| -                  |
+| `i32x4.ge_s`              |    `0x34`| -                  |
+| `i32x4.ge_u`              |    `0x35`| -                  |
+| -                         |    `0x36`| -                  |
+| -                         |    `0x37`| -                  |
+| -                         |    `0x38`| -                  |
+| -                         |    `0x39`| -                  |
+| -                         |    `0x3a`| -                  |
+| -                         |    `0x3b`| -                  |
+| -                         |    `0x3c`| -                  |
+| -                         |    `0x3d`| -                  |
+| -                         |    `0x3e`| -                  |
+| -                         |    `0x3f`| -                  |
+| `f32x4.eq`                |    `0x40`| -                  |
+| `f32x4.ne`                |    `0x41`| -                  |
+| `f32x4.lt`                |    `0x42`| -                  |
+| `f32x4.gt`                |    `0x43`| -                  |
+| `f32x4.le`                |    `0x44`| -                  |
+| `f32x4.ge`                |    `0x45`| -                  |
+| `f64x2.eq`                |    `0x46`| -                  |
+| `f64x2.ne`                |    `0x47`| -                  |
+| `f64x2.lt`                |    `0x48`| -                  |
+| `f64x2.gt`                |    `0x49`| -                  |
+| `f64x2.le`                |    `0x4a`| -                  |
+| `f64x2.ge`                |    `0x4b`| -                  |
+| `v128.and`                |    `0x4c`| -                  |
+| `v128.or`                 |    `0x4d`| -                  |
+| `v128.xor`                |    `0x4e`| -                  |
+| `v128.not`                |    `0x4f`| -                  |
+| `v128.bitselect`          |    `0x50`| -                  |
+| `i8x16.add`               |    `0x51`| -                  |
+| `i8x16.add_saturate_s`    |    `0x52`| -                  |
+| `i8x16.add_saturate_u`    |    `0x53`| -                  |
+| `i8x16.sub`               |    `0x54`| -                  |
+| `i8x16.sub_saturate_s`    |    `0x55`| -                  |
+| `i8x16.sub_saturate_u`    |    `0x56`| -                  |
+| `i8x16.mul`               |    `0x57`| -                  |
+| -                         |    `0x58`| -                  |
+| -                         |    `0x59`| -                  |
+| `i8x16.shl`               |    `0x5a`| -                  |
+| `i8x16.shr_s`             |    `0x5b`| -                  |
+| `i8x16.shr_u`             |    `0x5c`| -                  |
+| `i8x16.neg`               |    `0x5d`| -                  |
```

Comment:
Just noticed this while updating wabt: the MVP instructions group unary and binary ops together by type, but have unary before binary, e.g.:


|opcode|name|
|-|-|
|`0x67`|`i32.clz`|
|`0x68`|`i32.ctz`|
|`0x69`|`i32.popcnt`|
|`0x6a`|`i32.add`|
...